.travis.yml

```
@@ -2,7 +2,11 @@ sudo: required
 language: python
 python:
   - "3.6.4"
-# command to run tests
+services: 
+  - docker
+before_install: 
+  - docker build -t jr/easycrm -f Dockerfile .
 install:
   - pip install -r requirements.txt
-script: nosetests -v
+script:     
+  - docker run -e CI=true jr/easycrm nosetests -v -- --coverage --watchAll=false 
```

```suggestion
  - docker run -e CI=true jr/easycrm nosetests -v -- --coverage --watchAll=false 
```

We do also need to run docker with a port. 
@jasonlaw33 @ArnoWeichuXu Any suggestions of port we could use? format will looks like `-p  xx~xxxx:8090`

Another minor issue is that, we are no longer required to use `jr/` as docker image or container name prefix anymore.